content/docs/csidriver/installation/operator/powermax.md

```diff
@@ -36,6 +36,35 @@ Set up the iSCSI initiators as follows:
 
 For more information about configuring iSCSI, see [Dell Host Connectivity guide](https://www.delltechnologies.com/asset/zh-tw/products/storage/technical-support/docu5128.pdf).
 
+#### Linux multipathing requirements
+
+CSI Driver for Dell PowerMax supports Linux multipathing. Configure Linux multipathing before installing the CSI Driver.
+
+Set up Linux multipathing as follows:
```

Comment:
Change this sentence to read "Follow this procedure to set up Linux multipathing:

Reply:
done